pkg/oidcclient/login_test.go

```diff
@@ -712,6 +712,7 @@ func TestLogin(t *testing.T) { //nolint:gocyclo
 					h.stdinIsTTY = func() bool { return true }
 					require.NoError(t, WithClient(buildHTTPClientForPEM(formPostSuccessServerCA))(h))
 					require.NoError(t, WithSkipListen()(h))
+					h.skipBrowser = false // don't skip calling the following openURL func
```

Comment:
I'm surprised to see all of these `h.skipBrowser = false`, I would have expected this to be the default.

Reply:
Now I do see this.

```go
// Skip by default so we don't really open a browser. Each test can override this.
```